types/types.go

```diff
@@ -318,6 +350,23 @@ func (h BlockHeader) ID() BlockID {
 	return BlockID(HashBytes(buf))
 }
 
+// MinerOutputID returns the output ID of the miner payout.
+func (h BlockHeader) MinerOutputID() ElementID {
+	return ElementID{
+		Source: Hash256(h.ID()),
+		Index:  0,
+	}
+}
+
+// FoundationOutputID returns the output ID of the foundation payout. A Foundation subsidy output is only created every
+// 4380 blocks after the hardfork at block 298000.
+func (h BlockHeader) FoundationOutputID() ElementID {
+	return ElementID{
+		Source: Hash256(h.ID()),
+		Index:  1,
+	}
+}
```

Comment:
Not sure if I like these being methods of `BlockHeader`... We can revisit it later, I guess.